### ISSUE TITLE
Add. 14,667, foll. 41, 42 | https://bl.vuexistapps.us/manuscript/468 (Part6)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1023]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### General Notes

- In the content section, the folio number (Fol. 54) appears while the ms contain fol. 41, 42 and so, fol. 54 can not be a part of this ms.

**Comment by mikey-tang16** [comment]
Locus change to locus from="41" to="42" or simply locus from="41"? Other examples seem to suggest including only the lower fol. num.
@wlpotter

**Comment by wlpotter** [comment]
Yes, since we don't know from Wright the exact ending locus we are just providing the `@from` attribute. (Older records may have the `@to` estimated from where the subsequent ms item begins, but we stopped doing that later on in the project)

**Comment by wlpotter** [comment]
Although, here, we could include the `@to="42"` since there's only the two folios

**Comment by mikey-tang16** [comment]
locus from="41" to="42"
